pages/_document.tsx

```diff
@@ -10,6 +10,12 @@ class MyDocument extends Document {
     return (
       <Html lang="pt" className="antialiased p-0 mx-0 outline-none box-border">
         <Head>
+          <link
+            rel="preload"
+            href="/fonts/montserrat-v15-latin-600.woff2"
+            as="font"
+            crossOrigin=""
```

Comment:
Would be good to add a comment here explaining why the attribute has an empty string

Reply:
Sorry it don't have to be a empty string:

When you preload fonts, it’s important to set a crossorigin attribute even if the fetch is not cross-origin. You can see this in the code example from before:

While the reason why is complex, the end result is that browsers will ignore preloaded fonts without the crossorigin attribute, which will cause the browser to actually fetch the font twice (once during the preload, and then another time after discarding the preloaded font).

Refer to: https://wp-rocket.me/blog/font-preloading-best-practices/

I will add `crossorigin="anonymous"`

Reply:
I'm searching more about this attribute. I understand the behavior when we don't provide this attribute when preloading a font but it doesn't seem so easy to understand. 

I presumed that since it's the same document, wouldn't be a problem to preload the font without the `crossorigin` attribute.

Reply:
After searching for it a little bit:

https://datacadamia.com/web/html/crossorigin

`anonymous - CORS requests will have the mode set to same-origin.`

It's our case here since the fonts are being self-hosted and requested on the same document.

What happens when the ``cross-origin`` is not specified: 

![image](https://user-images.githubusercontent.com/48022589/99823580-55ef3a80-2b33-11eb-84c0-0017c6b8bf9f.png)